src/Popup/PopupInsert/PopupActionItem/index.js

```diff
@@ -10,7 +10,15 @@ import { Item, Action } from './styled'
  */
 class PopupActionItem extends Component {
   render () {
-    const { children, feature, title, disabled, onClick, style } = this.props
+    const { children, feature, title, disabled, onClick, style, href, target } = this.props
+
+    if (href && !disabled) {
+      return (
+        <Action role='button' onClick={(e) => onClick(e, feature)}>
+          {title ? <Link href={href} target={target || '_blank'}><span><Item style={style}>{title}</Item></span></Link> : children}
+        </Action>
+      )
+    }
```

Comment:
can this bit live in the `PopupActionNavigate` instead of the root `PopupActionItem`? @ntatko

Reply:
team agrees: this should live in it's own component: `PopupActionLink`